TemplateApplication/TemplateApplicationStandard.swift

```diff
@@ -25,35 +25,34 @@ actor TemplateApplicationStandard: Standard,
                                    EnvironmentAccessible,
                                    HealthKitConstraint,
                                    OnboardingConstraint,
-                                   AccountStorageConstraint,
                                    AccountNotifyConstraint {
     enum TemplateApplicationStandardError: Error {
         case userNotAuthenticatedYet
     }
 
-    private static var userCollection: CollectionReference {
+    static var userCollection: CollectionReference {
         Firestore.firestore().collection("users")
     }
 
-    @Dependency var accountStorage: FirestoreAccountStorage?
-
-    @AccountReference var account: Account
     @Application(\.logger) private var logger
-    
+
+    @Dependency(Account.self) private var account: Account? // optional, as Firebase might be disabled
+    @Dependency(FirebaseAccountService.self) private var accountService: FirebaseAccountService?
+
     
     private var userDocumentReference: DocumentReference {
```

Comment:
I would suggest to move the userDocumentReference into a separate module that we can use across views and other modules instead of having the standard to take care of that? I think this adds more trouble and a central reliance on the Standard than we need to. In ENGAGE-HF we separated it out in extension on the FIrestore types, not sure if we need to do that here or if a module serves as a better place for that?